cylc/flow/data_store_mgr.py

```diff
@@ -296,7 +296,7 @@ def initiate_data_model(self, reloaded=False):
         # Tidy and reassign task jobs after reload
         if reloaded:
             new_tasks = set(self.updates[TASK_PROXIES])
-            job_tasks = set(self.schd.job_pool.pool)
+            job_tasks = set(self.schd.job_pool.task_jobs)
```

Comment:
I had a look at the `job_pool.py` to confirm we had `.task_jobs` there as it is not added in this pull request. `JobPool` has both `pool` and `task_jobs`. So we were just using the wrong variable before?

Reply:
Exactly - Once I removed the scaffolding, a couple of errors cropped up (which is good), so fixed.